2996_reflection/reflection.md

```diff
@@ -369,37 +371,53 @@ template<typename... Ts> struct Tuple {
 template<typename... Ts>
   struct std::tuple_size<Tuple<Ts...>>: public integral_constant<size_t, sizeof...(Ts)> {};
 
-template<typename I, typename... Ts>
+template<std::size_t I, typename... Ts>
   struct std::tuple_element<I, Tuple<Ts...>> {
-    using type = [: template_arguments_of(^Tuple<Ts...>)[I] :];
+    using type = [: template_arguments_of(^Tuple<Ts...>)[I-1] :];
```

Comment:
Why -1? Presumably `template_arguments_of(^Tuple<int, double>)` gives you `{^int, ^double}` right?
```cpp
    using type = [: template_arguments_of(^Tuple<Ts...>)[I] :];
```
Although for that matter this is even better right:
```suggestion
    static constexpr std::array types = {^Ts...};
    using types = [: types[I] :];
```